docs/conf.py

```diff
@@ -20,227 +20,225 @@
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-sys.path.insert(0, os.path.abspath('..'))
+sys.path.insert(0, os.path.abspath(".."))
```

Comment:
arrg, I do prefer single quotes, but hey...

Reply:
There *is* an option for leaving quotes alone https://github.com/psf/black/issues/118#issuecomment-393298377, but that means that they are not auto-formatted when someone uses them differently.